### DISCUSSION TITLE
Adding a new element to the top

### DISCUSSION THREAD

**Discussion by osolmaz** [pradyunsg/furo#570] [Questions] [answered]
I want to add a "Star on GitHub" button to somewhere visible, preferably at top right of the page.

What would be the most ideal way to do that?

**Comment by pradyunsg** [comment, marked as answer]
There's no supported way to do this. You could hack around and inject your own HTML as described in https://pradyunsg.me/furo/customisation/injecting/

**Reply by osolmaz** [reply]
Thanks, I'll do that. It would still be awesome if there were an official way of doing it

> Controls which button is shown on the top of the page. The only supported values are "edit" (the default) and None.
> ```py
> html_theme_options = {
>     "top_of_page_button": "edit",
> }
> ```

Can we maybe extend this to include a "star on X" button? A call to action like this would significantly benefit open source projects that use Furo IMO. If it's aligned with Furo's principles, I can start working on a PR.
